src/pages/Systems/MachineInfo.tsx

```diff
@@ -83,10 +85,17 @@ const InfoItem = (props: { title: string; value: string; width?: string; loading
 };
 
 const IngestorInfo = () => {
+	const [opened, setOpened] = useState(false);
 	const [recentRecord] = useClusterStore((store) => store.currentMachineRecentRecord);
 	const [ingestorMachines] = useClusterStore((store) => store.ingestorMachines);
 	const ingestor = _.find(ingestorMachines, (ingestor) => ingestor.domain_name === recentRecord?.address);
 	const error = ingestor ? ingestor.error : null || null;
+	const deleteUrl = recentRecord?.address as string;
```

Comment:
You shouldn't use 'as' like this which is falsy. Fix the ts errors. Straight forward.